buildSrc/src/main/kotlin/smithy.module-conventions.gradle.kts

```diff
@@ -0,0 +1,5 @@
+plugins {
+    id("smithy.java-conventions")
+    id("smithy.integ-test-conventions")
```

Comment:
I dont think all modules need integ test conventions. I would prefer to isolate this to the 2 instances that need this.